integration/simulation/utils.go

```diff
@@ -12,9 +12,17 @@ import (
 	"github.com/obscuronet/obscuro-playground/go/log"
 )
 
-const testLogs = "../.build/simulations/"
+const (
+	envLogToStdout = "LOG_TO_STDOUT"
+	testLogs       = "../.build/simulations/"
+)
+
+func setupTestLog() {
+	_, isPresent := os.LookupEnv(envLogToStdout)
```

Comment:
Should probably check if the value is true/false rather than if the var it's set or not

Reply:
I chose checking for presence to avoid the situation where there's a misunderstanding about what this should be set to (e.g. CI sets it to "1", but we check for "true"). But that creates its own problems. Changed :)